docs/fastddsgen/pubsub_app/includes/first_app.rst

```diff
@@ -67,7 +67,7 @@ Then, compile the generated code executing the following commands from the *Fast
 .. code:: bash
 
     cd build
-    cmake ..
+    cmake <-DCOMPILE_EXAMPLES=ALL> ..
```

Comment:
This section is building the example autogenerated by Fast DDS-Gen. Therefore, it is not trying to explain how to build the Fast DDS examples in the repository.

Maybe, the [next steps](https://fast-dds.docs.eprosima.com/en/latest/fastddsgen/pubsub_app/pubsub_app.html?highlight=minimal%20functional%20example#summary-and-next-steps) section can be improved including instruction about how to build the examples.